packages/terra-clinical-item-display/translations/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "Terra.item-display.textStyleMeaningEnd": "{textStyleMeaning} end",
   "Terra.item-display.textStyleMeaningStrikethrough": "deletion",
-  "Terra.item-display.textStyleMeaningStrikethroughEnd": "deletion end"
+  "Terra.item-display.textStyleMeaningStrikethroughEnd": "deletion end",
+  "Terra.itemDisplay.comment": "Comment"
```

Comment:
is translation Jira Logged for other locales ?? if not logged yet we can make use exiting translation of clinical-result : https://github.com/cerner/terra-clinical/blob/c9f4512bf6e6bffd46ef4c42b53b50aad94486d3/packages/terra-clinical-result/translations/en.json#LL9C60-L9C67

Reply:
Yes, Jira has been logged for other locales, I just included all the files provided as part of the translation jira.
https://jira.cerner.com/browse/TRANS-14354